docs_src/custom_api_router/tutorial001.py

```diff
@@ -0,0 +1,158 @@
+"""
+File: tutorial001
+Author: prakash
+Created: 12/03/25.
+"""
+
+__author__ = "prakash"
+__date__ = "12/03/25"
+
```

Comment:
Can you remove all this metadata from the files? None of the files in the repository have this - contributions are tracked via the commit history 🙏